lineman/app/services/thread_query_service.coffee

```diff
@@ -52,15 +48,21 @@ angular.module('loomioApp').factory 'ThreadQueryService', (Records) ->
       parts = options.split ' '
       moment().startOf('day').subtract(parseInt(parts[0]), parts[1])
 
-    createFilter = (filter = 'show_all') ->
+    createFilter = (filters, queryType) ->
       (viewData) ->
         _.filter viewData, (thread) ->
-          return false if thread.isMuted() and filter != 'show_muted'
-          return false if thread.readerNotLoaded()
-          switch filter
-            when 'show_all'           then true
-            when 'show_muted'         then thread.isMuted()
-            when 'show_unread'        then thread.isUnread()
-            when 'show_participating' then thread.isParticipating()
-            when 'show_starred'       then thread.isStarred()
-            when 'show_proposals'     then thread.hasActiveProposal()
+
```

Comment:
Now threads get to run the gauntlet to see if they qualify for each thread collection.